babel/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -663,6 +663,69 @@ SqlNode InlineModOperator() :
     }
 }
 
+SqlNode CurrentTimeStampFunction() :
+{
+    final List<SqlNode> args = new ArrayList<SqlNode>();
+    SqlNode e;
+}
+{
+    <CURRENT_TIMESTAMP>
+    [
+        <LPAREN>
+        [
+            e = Expression(ExprContext.ACCEPT_SUB_QUERY) {
```

Comment:
Can we use AtomicRowExpression here instead of the more general Expression? Same for other functions as well.

Reply:
Actually I don't think we can. It looks like a case like SELECT current_time(x+y) fails when using AtomicRowExpression.

Reply:
CURRENT_TIME with a math expression as an input parameter doesn't make a whole lot of sense to me. Can you give a link to what test case this is covering?

Reply:
https://github.com/googleinterns/calcite/blob/f9ab6107abe6dee5f8aa27239d41c0717bf43366/core/src/test/java/org/apache/calcite/sql/parser/SqlParserTest.java#L4208

Reply:
Thanks. That seems like a bizarre test case. All of the docs I've checked don't show support for that syntax.

Nevertheless, since the test already exists, we might as well support it.